components/TodaysCards/index.js

```diff
@@ -9,19 +8,20 @@ const StyledList = styled.ul`
 
 export default function TodaysCards({ events, onToggleAlarm }) {
   return (
-    <>
-      <h1>Today&#96;s Events</h1>
-      <StyledList>
-        {events.map((event) => {
+    <StyledList>
+      {events.length === 0 ? (
+        <h1>No events for today</h1>
+      ) : (
+        events.map((event) => {
           return (
             <TodaysCard
-              key={uid()}
+              key={event.id}
               event={event}
               onToggleAlarm={onToggleAlarm}
             ></TodaysCard>
```

Comment:
This can be self closing if there are no `children`.

```suggestion
            />
```